lib/node_modules/@stdlib/assert/has-same-constructor/README.md

```diff
@@ -0,0 +1,93 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2018 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# hasSameConstructor
+
+> Test if two values have same constructor.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var isArrayLikeObject = require( '@stdlib/assert/has-same-constructor' );
+```
+
+#### hasSameConstructor( value , value )
+
+Tests if two values have same `constructor`.
+
+<!-- eslint-disable object-curly-newline -->
+
+```javascript
+    > var obj1 = new Date();
+      var obj2 = new Date();
+      console.log(hasSameConstructor(obj1, obj2));
+    true
+
+    > var obj1 = new Date();
+      var obj3 = new String("Hello");
+      console.log(hasSameConstructor(obj1, obj3));
+    false
+
+    > var obj3 = new String("Hello");
+      var obj4 = "World";
+      console.log(hasSameConstructor(obj3, obj4));
+    true
```

Comment:
Please see other packages for how we format examples. This does not match project conventions.